dask_cuda/__init__.py

```diff
@@ -1,6 +1,16 @@
+import dask
+import dask.dataframe.shuffle
+from .explicit_comms.dataframe_shuffle import rearrange_by_column_tasks_wrapper
 from ._version import get_versions
 from .cuda_worker import CUDAWorker
 from .local_cuda_cluster import LocalCUDACluster
 
 __version__ = get_versions()["version"]
 del get_versions
+
+
+# Monkey patching Dask to make use of explicit-comms
+dask.dataframe.shuffle.rearrange_by_column_task_org = (
+    dask.dataframe.shuffle.rearrange_by_column_tasks
+)
+dask.dataframe.shuffle.rearrange_by_column_tasks = rearrange_by_column_tasks_wrapper
```

Comment:
Sorry @rjzamora , I'm not sure if I follow your comment. Are you saying that there's already check for `dask.config.get("explicit-comms", False)` somewhere, or are you asking if that's what I'm proposing? To be clear, that is indeed what I'm proposing, if it already exists, could you point me to that?

I'm proposing we do that because this seems to otherwise be effectively changing the default Dask behavior, and any unaware user simply running `importing dask_cuda` would have that happening implicitly.

Reply:
Sorry for not clarifying - This is already in the code. [See this comment](https://github.com/rapidsai/dask-cuda/pull/494/files#r567867614).

Reply:
I have clean up the monkey patch -- we replace `rearrange_by_column_tasks` with a wrapper function always **but** it simply pass-through `*args, **kwargs` when `DASK_EXPLICIT_COMMS=False`.

Reply:
Indeed, sorry I totally missed that in the implementation. Thanks Rick for clarifying and Mads for the update there too!